### PR TITLE
Feat/add complete transaction inscription start

### PR DESCRIPTION
**Pull request by TheYakuza**
add inscription file and base to complete transaction

### CODE REVIEW

File: lib/transaccion_completa/Inscription.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            $body = json_decode($httpResponse->getBody(), true);
+            if (isset($body["error_message"])) {
+                $tbkErrorMessage = $body["error_message"];
+                $message = "$message. Details: $tbkErrorMessage";
+            }
+            throw new InscriptionStartException($message, $httpCode);
+        }
+        $responseJson = json_decode($httpResponse->getBody(), true);
+
+        if (isset($responseJson["error_message"])) {
```
Comment (eugenioLopezRamos) [inline lib/transaccion_completa/Inscription.php]: This excerpt was removed from the rest of the transactions of the SDK, is it correct for this to be here?

Comment (TheYakuza) [reply lib/transaccion_completa/Inscription.php]: fixed at 1eacd66

File: lib/transaccion_completa/Options.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    public function __construct($apiKey, $commerceCode)
+    {
+        $this->setApiKey($apiKey);
+        $this->setCommerceCode($commerceCode);
+    }
+
+    public static function defaultConfig()
+    {
+        return new Options(self::DEFAULT_API_KEY, self::DEFAULT_COMMERCE_CODE);
```
Comment (eugenioLopezRamos) [inline lib/transaccion_completa/Options.php]: I believe these params should be whatever specific ones are used for TransaccionCompleta, or are they the ones here?

File: lib/transaccion_completa/exceptions/Exceptions.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ * @category
+ * @package Transbank\TransaccionCompleta\Exceptions
+ *
+ */
+
+
+namespace Transbank\TransaccionCompleta;
+
+
+class Exceptions extends \Exception
```
Comment (eugenioLopezRamos) [inline lib/transaccion_completa/exceptions/Exceptions.php]: Is it not more alike the rest of the SDK to name this `TransaccionCompletaException`?

File: lib/transaccion_completa/inscription/InscriptionFinishResponse.php [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    public $lastFourCardDigits;
+
+    public function __construct($json)
+    {
+        $responseCode = isset($json["response_code"]) ? $json["response_code"] : null;
+        $tbkUser = isset($json["tbk_user"]) ? $json["tbk_user"] : null;
+        $authorizationCode = isset($json["authorization_code"]) ? $json["authorization_code"] : null;
+        $creditCardType = isset($json["credit_card_type"]) ? $json["credit_card_type"] : null;
+        $lastFourCardDigits = isset($json["last_four_card_digits"]) ? $json["last_four_card_digits"] : null;
+        $this->setResponseCode($responseCode);
```
Comment (eugenioLopezRamos) [inline lib/transaccion_completa/inscription/InscriptionFinishResponse.php]: why not
```
$variable = yadayayada;
$this->setVariable($variable);
```
instead?

File: lib/transaccion_completa/exceptions/Exceptions.php (hunk 3 above)
Comment (TheYakuza) [reply lib/transaccion_completa/exceptions/Exceptions.php]: fixed at f12e071

File: lib/transaccion_completa/inscription/InscriptionFinishResponse.php (hunk 4 above)
Comment (TheYakuza) [reply lib/transaccion_completa/inscription/InscriptionFinishResponse.php]: You're right is more consistent with the others SDK like the way you are writing 
fixed at 0e6955a

**Comment by TheYakuza** [comment]
I've change the target branch to feat7add-rest-supppot

File: lib/onepay/exceptions/RefundCreateException.php [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
 namespace Transbank\Onepay\Exceptions;
 /**
- * 
+ *
  * class RefundCreateException
  * Model object for raising exceptions when a Refund::create transaction fails.
- * 
- * @package Transbank\Onepay\Exceptions
+ *
+ * @package Transbank\Onepay\TransaccionCompletaException
```
Comment (eugenioLopezRamos) [inline lib/onepay/exceptions/RefundCreateException.php]: Stop using replace all on the whole project, you can filter by directory

File: init.php [hunk 6]
```diff
@@ -27,7 +27,7 @@
 require_once(dirname(__FILE__) . '/lib/onepay/RefundCreateRequest.php');
 require_once(dirname(__FILE__) . '/lib/onepay/RefundCreateResponse.php');
 
-// Exceptions
+// TransaccionCompletaException
```
Comment (eugenioLopezRamos) [inline init.php]: This is wrong

File: lib/onepay/exceptions/RefundCreateException.php (hunk 5 above)
Comment (TheYakuza) [reply lib/onepay/exceptions/RefundCreateException.php]: Yeah, I've noticed that, now is fixed at f29d06c

File: init.php (hunk 6 above)
Comment (TheYakuza) [reply init.php]: fixed at f29d06c

File: lib/transaccion_completa/Options.php [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+ *
+ */
+
+
+namespace Transbank\TransaccionCompleta;
+
+
+class Options
+{
+    const DEFAULT_API_KEY = '';
```
Comment (eugenioLopezRamos) [inline lib/transaccion_completa/Options.php]: Do we still not have these, or is this just an oversight?

Comment (TheYakuza) [reply lib/transaccion_completa/Options.php]: No, we don't have the default yet

**Pull request merged**